protos/bottle/account/v1/user.proto

```diff
@@ -21,4 +21,6 @@ message User {
 
   AccountType account_type = 7;
   string company_name = 8;
+
+  bool tax_exempt = 9;
```

Comment:
This maps out 1-1 with a db column. Talking with carl though, this will probably get updated in the future. I'm wondering for future proofing this a bit more, do we want to attach it to the order instead? Then the other services don't have to worry about where it comes from.

Reply:
I think this is better on the order for now :+1: